### DISCUSSION TITLE
License Refund

### DISCUSSION THREAD

**Discussion by refundboy** [aclap-dev/video-downloadhelper#1030] [License & Purchase requests]
### What's your issue?

Hi, bought the license to try to download a few videos. They are scrambled and cannot be viewed.

### The 10 first digits (only 10) of your license number or email address

aeb9561ce5

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
Please be more specific. Share your versions and if possible a public link to a video with this issue.

**Reply by refundboy** [reply]
Hi I was trying to download videos from onlyfans, they turned out to be scrambled 
Version 9.0.2.12 (stable)

**Reply by jcfrog** [reply]
I heard there was some DRM on onlyfans. Do you have DRM on your video?

**Reply by refundboy** [reply]
It seems like DRM is the culprit :(
The Videos were all scrambled. 
This was the only reason I paid for the license

**Reply by jcfrog** [reply]
If you are not satisfied you have one month to ask for a full refund.

**Reply by refundboy** [reply]
Thank you so much, may I have the refund at your earliest convenience?

**Reply by jcfrog** [reply]
You have been refunded. Should be available in a few business days.
